docs/multicluster/upgrade.md

```diff
@@ -0,0 +1,49 @@
+# Antrea Multi-cluster Upgrade Guide
+
+The Antrea Multi-cluster feature is introduced from v1.5.0. There is no data-plane related changes in release v1.5.0, so Antrea deployment and Antrea Multi-cluster deployment are indenpendent. Please refer to the section `Antrea upgrade and supported version skew` in [Antrea Versioning](../versioning.md) to learn the requirement of Antrea upgrade. In this doc, it focuses on Multi-cluster deployment only.
```

Comment:
Out of curiosity, why would Antrea deployment and Antrea Multi-cluster deployment become dependent with each other if there are dataplane changes?

Reply:
yeah, they will be dependent. I think I need to try upgrade test and refine this doc later.